.happy/terraform/modules/batch/main.tf

```diff
@@ -101,6 +101,54 @@ resource aws_batch_job_definition cxg_job_def {
 })
 }
 
+resource aws_batch_job_definition atac_job_def {
+  type = "container"
+  name = "dp-${var.deployment_stage}-${var.custom_stack_name}-atac"
+  container_properties = jsonencode({
+  "jobRoleArn": "${var.batch_role_arn}",
+  "image": "${var.image}",
+  "memory": 64000,
```

Comment:
I recommend using less than this. If you use 64000, it might not fit in a 64G machine and will require a bigger type, due to the OS and the container overhead. Using 60k-62k is a safer option.